docs/src/single-validator-manager.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Single Validator Stake Pool Manager Program
+---
+
+A work-in-progress program for permissionless management of stake pools that are
+only delegated to one validator.
+
+## Overview
+
+The single-validator manager program defines a canonical stake pool for any
+validator on the network, which can be created by anyone.
+
+The program manages the "delegation strategy" of the pool, which is to maximize
+the amount of stake on one validator, while maintaining a small amount of SOL
+in the reserve for small stakers.
+
+## Fees
+
+Pools are configured with a SOL deposit fee of 8 basis points, which corresponds
```

Comment:
It'd be really nice to tie this to the current epoch inflation percentage. As inflation trends towards its terminal value, 8bps will be too much. Unfortunately there's no `Inflation` sysvar currently or another way to figure inflation from within the runtime

Reply:
Yeah that would definitely be the best option. I would be happy to add the sysvar, since it could also be used to calculate staking rewards on-chain in case we ever add rewards claiming :eyes: